youtube_dl/extractor/pbs.py

```diff
@@ -232,6 +247,12 @@ def _real_extract(self, url):
                 'url': closed_captions_url,
             }]
 
+        # video.pbs.org video.pbs.org/videoInfo/... frequently provides an obscure 'title' value, like
+        # 'Full Episode', 'Episode 5', etc. prepend program->title
+        alt_title = info.get('program', {}).get('title')
+        if alt_title:
+            info['title'] = alt_title + ' - ' + re.sub(r'^' + alt_title + '[\s\-\:]+', '', info['title'])
```

Comment:
Wouldn't it be easier to just use `info['title']` when it contains `alt_title` instead of cutting it from `info['title']` and then prepending again?